arch/arm/core/cortex_m/mpu/arm_mpu.c

```diff
@@ -109,18 +109,18 @@ static inline u32_t _get_region_attr_by_type(u32_t type, u32_t size)
 	switch (type) {
 #ifdef CONFIG_USERSPACE
 	case THREAD_STACK_REGION:
-		return _get_region_attr(1, P_RW_U_RW, 0, 1, 0,
```

Comment:
According to http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.dui0553a/Cihegaib.html you are setting here "Outer and inner write-back. No write allocate". You will achieve better performance with "Outer and inner write-back. Write and read allocate"

Reply:
Indeed, it seems WBWA is used on the RAM region when the MPU is not enabled. I'll change that.

Reply:
agree with @pizi-nordic. I made a similar comment in #8185 about configuring RAM regions as Write-Back Write-Allocate (and Non-Shareable).